app/src/main/java/com/example/marvelapp/database/mapper/CharacterDataBaseMapper.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.marvelapp.database.mapper
+
+import com.example.marvelapp.database.entity.CharacterEntity
+import com.example.marvelapp.entity.Character
+
+class CharacterDataBaseMapper : BaseMapper<CharacterEntity, Character> {
```

Comment:
I think that this is not necessary, you can use extension functions as you did in the service response

Reply:
done